Copy my past project

AndreaM2429 opened the pull request:
I copy the code from my second trial.
This is a Landing page and it's a very simple page.

AndreaM2429 committed:
Copy my past project

AndreaM2429 reviewed (commented):
Hello!
🎉 Good job so far! 💯 You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.

# Highlights
Descriptive PR ✔️
Correct GitHub flow ✔️

# Required Changes 🛠
- _♻Please review the comments unther this review for the specific changes._ 

# Optional Suggestions 💡
- NA

usorfaitheloho reviewed (changes requested):
> Hi @AndreaM2429  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! ![](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️

- [ ] It is code review best practice using inline comments to highlight things that could be done better in a code or group of codes, this should be done in a polite way and can be made more engaging by adding emojis 😄 . You can check this [article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) for more details

> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AndreaM2429 commented:
Hello @usorfaitheloho
Do I need to add some emojis in each comment?

usorfaitheloho commented:
Yes you should please @AndreaM2429

AndreaM2429 commented:
Done! I edited all comments

aamirkhan2478 reviewed (changes requested):
Hi @AndreaM2429 , 

![](https://media3.giphy.com/media/RcZlXVml8oKtoGQeqe/giphy.gif?cid=ecf05e47cw5pjd1gapdzk1t3vp0n73caujoqxw3vkcu8ssng&rid=giphy.gif&ct=g)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.

 ## Required Changes ♻️

- [ ] As mentioned by the previous reviewer :point_up_2: please use  `inline comments` to highlight things that could be done better in a code or group of codes, this should be done in a polite way but still you are not using any `inline comments` check this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) For how to use the `inline comments` GitHub feature :slightly_smiling_face: 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AndreaM2429 commented:
Hello @aamirkhan2478 
I think this is an inline comment, is'n it?
![image](https://user-images.githubusercontent.com/119966922/235719321-994ac1c9-e025-4000-9e6b-7a27e7d1605f.png)

AndreaM2429 commented on index.html:
```diff
@@ -0,0 +1,34 @@
+        <div id="video2">
+            <iframe id="video" src="https://www.youtube.com/embed/BFqtfxcNEXM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+        <div>
+            <form action="https://www.freecodecamp.com/email-submit" id="form">
+                <input type="email" id="email" placeholder="Email">
+                <input type="submit" id="submit">
+            </form>
+        </div>
```
I think is better to use a `<section>`instead to `<div>` this is according to the best practices in HTML code 🤓.

According the best practices in HTML code, the browsers use some tags for de SEO and the page positioning.  There are a lot of diferent tags, check [here](http://diveintohtml5.it/semantics.html) for more information about them.

AndreaM2429 commented on index.html:
```diff
@@ -0,0 +1,34 @@
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    </head>
+    <header id="header">
+        <img src="https://consumer.huawei.com/content/dam/huawei-cbg-site/common/mkt/plp/phones/new-phones/model-banner/img-0904/mate50.png" width="150px" alt="img" id="header-img">
+        <h1>HUAWEI LEATEST TOP PRODUCTS</h1>
+        <nav id="nav-bar">
+            <a rel="stylesheet" href="#video">Pricing</a>
+            <a rel="stylesheet" href="">Email list</a>
+            <a rel="stylesheet" href="">How it works</a>
+        </nav>
+    </header>
```
💭I suggest to put the `<header>` inside the `<body>` accordig with the best practices in HTML code.

All the parts of a web page such as `header`, `section`,` footer`, have to be inside the body because they are the elements that the user could see in the screen. ✔✔
The only part that has to be out of the body is the head, that contains metadata (information about the page) and the DOCTYPE (is the HTML standar).

AndreaM2429 reviewed (commented):
Hello!
🎉 Good job so far! 💯 You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.

# Highlights
Descriptive PR ✔️
Correct GitHub flow ✔️

# Required Changes 🛠
- _♻Please review the comments unther this review for the specific changes._ 

# Optional Suggestions 💡
- NA

AndreaM2429 commented:
I made this review @aamirkhan2478 
I think is correct now, isn't it?

aamirkhan2478 commented:
> I made this review @aamirkhan2478 I think is correct now, isn't it?

Hi @AndreaM2429 
Yes this is correct now but please explain more  like `why` you give these required changes okay

AndreaM2429 commented:
I added more details about the changes @aamirkhan2478
Are there ok? 
is it ok that the review isn't in the top of the page? Because I deleted the first rewiev and made another so it automatically positioning behind the other comments.

AndreaM2429 commented:
> I added more details about the changes @aamirkhan2478 Are there ok? is it ok that the review isn't in the top of the page? Because I deleted the first rewiev and made another so it automatically positioning behind the other comments.

Hello @aamirkhan2478 
Could you answer my past question please? 🙏

aamirkhan2478 commented:
> > I added more details about the changes @aamirkhan2478 Are there ok? is it ok that the review isn't in the top of the page? Because I deleted the first rewiev and made another so it automatically positioning behind the other comments.
> 
> Hello @aamirkhan2478 
> Could you answer my past question please? 🙏

@AndreaM2429 
Yes it's okay good :+1: now submit for review again

divinecharlotte reviewed (approved):
# Hello @AndreaM2429   😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Good implementation of required changes👍 





## Congratulations! 🎉 

<img src="https://media3.giphy.com/media/3KC2jD2QcBOSc/giphy.gif"/>

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
